modules/services/polybar.nix

```diff
@@ -211,7 +211,7 @@ in {
 
       Service = {
         Type = "forking";
-        Environment = "PATH=${cfg.package}/bin:/run/wrappers/bin";
+        Environment = "";
```

Comment:
If the `PATH` is no longer necessary, then why keep `Environment = ""`?

Reply:
Sure I can remove it altogether. Would I have to open a new pull request??